static/css/style.css

```diff
@@ -1116,6 +1116,12 @@ div#container-display-show {
     width: 300px;
 }
 
+#show-selector {
+    width: auto;
+    height:25px;
+    padding:1px;
+}
+
```

Comment:
you added the id `show-selector`. that doesn't exist. You should modify the existing `select#select-show`. then it works fine.